.changeset/generator-components.md

```diff
@@ -0,0 +1,5 @@
+---
+"@asyncapi/generator-components": major
```

Comment:
```suggestion
"@asyncapi/generator-components": minor
```

this is minor release 

have a look later at https://semver.org/

tl;dr major is when breaking changes are introduced